linalg/vector.hpp

```diff
@@ -281,6 +281,7 @@ class Vector
    /// v = median(v,lo,hi) entrywise.  Implementation assumes lo <= hi.
    void median(const Vector &lo, const Vector &hi);
 
+   /// Extract entries listed in `dofs` to the output `elemvect`
```

Comment:
Perhaps group these two methods with `///@{` and `///@}` to reuse the description on both or just comment both separately.